src/components/events/ReportModal.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { XIcon } from "../icons/Icons";
+
+export default function ReportModal({ title, onDismiss }) {
+  const [value, setValue] = useState(null)
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Once the api is written, I'm thinking the url will be something like a POST request: /api/deal/report/, taking a body that includes the deal Object at a minimum that's being reported. With a reason.